Extension/src/LanguageServer/configurations.ts

```diff
@@ -436,9 +436,13 @@ export class CppProperties {
     private async readNodeAddonIncludeLocations(rootPath: string): Promise<void> {
         let error: Error | undefined;
         let pdjFound: boolean = false;
-        const package_json: any = await fs.promises.readFile(path.join(rootPath, "package.json"), "utf8")
-            .then(pdj => {pdjFound = true; return JSON.parse(pdj); })
-            .catch(e => (error = e));
+        const package_json: any = await fs.promises.readFile(path.join(rootPath, "package.json"), "utf8");
+        try {
+            pdjFound = true;
+            return JSON.parse(package_json);
```

Comment:
This `return` should actually be the assignment to the `package_json` variable

Reply:
And can you rename `package_json` -> `packageJson` while you're fixing this code?

Reply:
please check the modified code.